Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/BankSelectorTokenizationViewModel.swift

```diff
@@ -193,12 +193,20 @@ class BankSelectorTokenizationViewModel: ExternalPaymentMethodTokenizationViewMo
         
     @objc
     override func startTokenizationFlow() {
-        super.startTokenizationFlow()
```

Comment:
Don't we need `super` here anymore?

Reply:
We need to override cause there're a couple of steps that have to be done before the steps that are done in `super`.